File: ft_webview/src/main/java/spa/lyh/cn/ft_webview/webview/base/BaseActivity.java
package spa.lyh.cn.ft_webview.webview.base;

import android.content.Context;

import androidx.appcompat.app.AppCompatActivity;

import spa.lyh.cn.ft_webview.webview.util.LanguageUtils;

public class BaseActivity extends AppCompatActivity {

    @Override
    protected void attachBaseContext(Context newBase) {
        // 国际化适配（绑定语种）
        if (LanguageUtils.isActivited()){
            super.attachBaseContext(LanguageUtils.getLanguageContext(newBase));
        }else {
            super.attachBaseContext(newBase);
        }
    }
}

File: ft_webview/src/main/java/spa/lyh/cn/ft_webview/webview/base/BaseActivity.kt
package spa.lyh.cn.ft_webview.webview.base

import android.content.Context
import androidx.appcompat.app.AppCompatActivity
import spa.lyh.cn.ft_webview.webview.util.LanguageUtils

open class BaseActivity:AppCompatActivity(){

    override fun attachBaseContext(newBase: Context?) {
        if(LanguageUtils.isActivited()){
            super.attachBaseContext(LanguageUtils.attach(newBase))
        }else{
            super.attachBaseContext(newBase)
        }
    }
}